src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveEvaluator.cs

```diff
@@ -90,11 +91,17 @@ public abstract class InteractiveEvaluator : IInteractiveEvaluator, ICurrentWork
             _commandsFactory = commandsFactory;
             _commands = commands;
 
-            var hostPath = interactiveHostPath;
-            _interactiveHost = new InteractiveHost(replType, hostPath, initialWorkingDirectory);
-            _interactiveHost.ProcessStarting += ProcessStarting;
-
+            // The following settings will apply when the REPL starts without .rsp file.
+            // They are discarded once the REPL is reset.
+            ReferenceSearchPaths = ImmutableArray<string>.Empty;
```

Comment:
These are just default values for an edge case scenario, when the .rsp file is actually deleted. Whenever the REPL is restarted via #reset command a different code path sets up the context.